frontend/src/static/css/v_authorship.scss

```diff
@@ -136,6 +101,18 @@
         margin-left: .5rem;
         text-decoration: none;
       }
+
+      .icons {
+        margin-right: 8px;
+        vertical-align: middle;
+      }
+
+      .fileTypeLabel {
+        border-radius: 5px;
+        display: inline-block;
+        font-weight: bold;
```

Comment:
The font-weight bold is reserved for first-timer issue

Reply:
I think what prof meant was to leave the removal of `font-weight: bold;` as a first timer issue 😂